e2e/test-component/scenarios/embedding-sdk/editable-dashboard.cy.spec.tsx

```diff
@@ -45,4 +45,65 @@ describeEE("scenarios > embedding-sdk > editable-dashboard", () => {
     cy.findByRole("tab", { name: "Overview" }).should("not.exist");
     cy.findByRole("tab", { name: "History" }).should("not.exist");
   });
+
+  describe("loading behavior for both entity IDs and number IDs (metabase#49581)", () => {
```

Comment:
All component tests use the same structure.

2 success cases for entity and number ID, and 3 failure cases (2 for entity ID, 1 for number).